example/package.json

```diff
@@ -26,6 +27,7 @@
     "@babel/runtime": "^7.12.5",
     "babel-loader": "^8.2.1",
     "metro-react-native-babel-preset": "^0.59.0",
+    "rimraf": "^3.0.2",
```

Comment:
I can revert the rimraf change, if so desired. As there still exists a problem here. 
I could add a `clean:windows` in addition to `clean`  but sadly there is no way I could find to conditionally run postinstall or a different postinstall on windows, without resorting to a fully external script.js

yarn install will always fail because postinstall will fail on windows because of lack of support for `yarn pod-update`
So for anyone wanting to iterate/test on windows, that will need to be manually modified and not checked in.
Unless there is a better suggested way to not break existing workflows.

My main goal was to not break any existing flows at this time, and figured if anyone was more interested in a seamless windows dev loop, they can investigate in that future. We have been using an similiar implementation of this for over a year and not need any changes, so I figured windows dev loop is pretty low pri.